README.md

```diff
@@ -128,11 +128,19 @@ On an ALP system where the Ansible workload container has been installed,
 using the `install` runlabel, the examples will be available under the
 `/usr/local/share/ansible-container/examples/ansible` directory.
 
-There are three playbooks currently under /usr/local/share/ansible-container/examples/ansible
+There are fourteen playbooks currently under /usr/local/share/ansible-container/examples/ansible
 * playbook.yml
 * network.yml
 * setup_libvirt_host.yml
-
+* create_alp_vm.yml
```

Comment:
Have you validated that the create_alp_vm and create_tumbleweed_vm playbooks are working still... At the time that I wrote the original playbooks I created the alp variant because the tumbleweed one wasn't working, which eventually turned out to be due to a change to the tumbleweed image build that required a tweak to one of the associated the virt-install parameters.